WooCommerce/Classes/Analytics/WooAnalytics.swift

```diff
@@ -234,7 +234,7 @@ private extension WooAnalytics {
             }
         }
 
-        return ["widgets": widgetAnalyticNames]
+        return ["widgets": widgetAnalyticNames.description]
```

Comment:
This will result in `["today-stats", "week-stats"]`. Is it expected to have braces and quotation marks?
What about `widgetAnalyticNames.joined(separator: ",")` to have `today-stats,week-stats`?

Do we have other examples of analytics events where arrays of strings are stored and later processed? I wasn't able to quickly find any reference.

Reply:
Good catch, lets switch to `joined(separator: ",")` we do the same for add-ons